addons/event/static/src/template_reference_field/field_event_mail_template_reference.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<templates xml:space="preserve">
+
+    <t t-name="event.mailTemplateReferenceField" t-inherit="web.ReferenceField" t-inherit-mode="primary">
+        <xpath expr="//t[@t-if='!props.readonly and !hideModelSelector']" position="after">
+            <t t-elif="getValue()">
+                <div class="d-flex justify-content-center flex-grow-0 flex-shrink-0" style="width: 1.5rem !important; max-width: 1.5rem !important; margin-right: 0.5rem;">
```

Comment:
Can we use a css file for the component rather than inline rules?
If we're choosing to go for a full blown Component, might as well go all in.